utils/make-packages.sh

```diff
@@ -55,7 +65,10 @@ EOF
 			cat <<EOF
   * Felix ${version} (from Git commit ${sha}).
 EOF
-			git show ${version} --format=oneline -s | head -n -1 | tail -n +5 | sed 's/^/    /'
+			for tag in ${tags}; do
+			    echo "    [Changes recorded in $tag tag]"
```

Comment:
Slightly concerned about the square brackets, could they end up being some sort of markup?  What format are those text blocks in, are they just plain text?